libs/ui-watt/src/lib/components/dropdown/watt-dropdown.component.scss

```diff
@@ -49,6 +53,16 @@ watt-dropdown {
   .mat-select.mat-select-invalid.ng-pristine .mat-select-arrow {
     color: currentColor;
   }
+
+  .watt-chip-label {
+    margin-left: var(--watt-space-s);
+    background: var(--watt-color-neutral-white);
+  }
+
+  .chip {
+    opacity: 0;
```

Comment:
Question: if should be hidden, why not use display:none ?

Reply:
Because `display: none` would make the dropdown not clickable, and then we would need to open the dropdown programmatically.

Reply:
And visibility:hidden is not an option i guess? :D

Reply:
It has now been changed for `visibility: hidden` :D